test/Interpreter/objc_implementation_objc_client.m

```diff
@@ -0,0 +1,114 @@
+//
```

Comment:
This test is currently failing in iOS simulators:

```
/Users/ec2-user/jenkins/workspace/swift-PR-macos/branch-main/swift/test/Interpreter/objc_implementation_objc_client.m:76:12: error: CHECK: expected string not found in input
 // CHECK: ImplClass.someMethod()
           ^
<stdin>:10:18: note: scanning from here
implProperty = 42
                 ^
<stdin>:18:53: note: possible intended match here
 2 CoreFoundation 0x00007fff2040a121 +[NSObject(NSObject) instanceMethodSignatureForSelector:] + 0
                                                    ^

Input file: <stdin>
Check file: /Users/ec2-user/jenkins/workspace/swift-PR-macos/branch-main/swift/test/Interpreter/objc_implementation_objc_client.m

-dump-input=help explains the following input dump.

Input was:
<<<<<<
            .
            .
            .
            5: someMethod = SwiftSubclass.someMethod() 
            6: implProperty = 0 
            7: implProperty = 42 
            8: otherProperty = 1 
            9: otherProperty = 13 
           10: implProperty = 42 
check:76'0                      X error: no match found
           11: SwiftSubclass It's better to burn out than to fade away. 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           12: 2022-10-06 00:17:59.166 objc_implementation_objc_client[10405:1972293] -[Swift.__SharedStringStorage dataUsingEncoding:]: unrecognized selector sent to instance 0x600003e08000 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           13: 2022-10-06 00:17:59.176 objc_implementation_objc_client[10405:1972293] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[Swift.__SharedStringStorage dataUsingEncoding:]: unrecognized selector sent to instance 0x600003e08000' 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           14: *** First throw call stack: 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           15: ( 
check:76'0     ~~
           16:  0 CoreFoundation 0x00007fff203fb4b4 __exceptionPreprocess + 242 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           17:  1 libobjc.A.dylib 0x00007fff2019ebe7 objc_exception_throw + 48 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           18:  2 CoreFoundation 0x00007fff2040a121 +[NSObject(NSObject) instanceMethodSignatureForSelector:] + 0 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
check:76'1                                                         ?                                               possible intended match
           19:  3 CoreFoundation 0x00007fff203ff9bc ___forwarding___ + 1433 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           20:  4 CoreFoundation 0x00007fff20401ae8 _CF_forwarding_prep_0 + 120 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           21:  5 objc_implementation_objc_client 0x0000000105cc54d2 print + 50 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           22:  6 objc_implementation_objc_client 0x0000000105cc530f main + 127 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           23:  7 dyld 0x0000000105cd9e1e start_sim + 10 
check:76'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            .
            .
            .
>>>>>>
```

Reply:
I haven't been able to reproduce this locally. Disabling outside of macosx for now.